File: example-pycharm-refactor/Movie.py
import string

CHILDRENS = 2
REGULAR = 0
NEW_RELEASE = 1

class Movie:
    _title: string = None
    _priceCode: int = None

    def __init__(self, title: string, priceCode: int):
        self._title = title
        self._priceCode = priceCode

    def getPriceCode(self):
        return self._priceCode

    def setPriceCode(self, arg: int):
        self._priceCode = arg

    def getTitle(self):
        return self._title

File: exemplo-vscode/Movie.ts
export default class Movie {

    public static CHILDRENS = 2 as const;
    public static REGULAR = 0 as const;
    public static NEW_RELEASE = 1 as const;

    private _title: string;
    private _priceCode: number;

    public Movie(title: string, priceCode: number) {
        this._title = title;
        this._priceCode = priceCode;
    }

    public getPriceCode(): number {
        return this._priceCode;
    }

    public setPriceCode(arg: number): void {
        this._priceCode = arg;
    }

    public getTitle(): string {
        return this._title;
    }
}